blocks/into/__title/into__title.css

```diff
@@ -0,0 +1,37 @@
+.into__title{
+    display: flex;
+    width: 300px;
+    height: 174px;
+    flex-direction: column;
+    justify-content: center;
+    flex-shrink: 0;
```

Comment:
Almost everywhere you've used `flex-shrink: 0;`, it will be great to remove that.